ProcessOptimizer/model_systems/model_system.py

```diff
@@ -8,29 +11,20 @@ class ModelSystem:
 
     Parameters:
     * `score` [callable]:
-        Function for calculating the score of the system at a given point in 
-        the parameter space. It is expected to have the following signature:
-
-            def score(x, rng=np.random.default_rng(), noise_std=noise_value):
-                # * x is the point in the parameter space where the score will 
-                #   be evaluated. Note that x is in the original coordinates of
-                #   the parameter space, not normalized coordinates. 
-                # * rng is the random number generator used for adding noise 
-                #   to the system. The user can set a seed through this 
-                #   parameter. There must be a default value for this parameter. 
-                # * noise_std is the standard deviation of the noise. There
-                #   must be a default value for this parameter. 
-                ....
-                # The score of the system is returned
-                return score
+        Function for calculating the noiseless score of the system at a given point in 
+        the parameter space. It is expected to have the following signature.
 
     * `space` [Space]:
         The parameter space in the form of a Space object. 
 
     * `true_min` [float]:
-        The true minimum value of the score function within the parameter space. 
+        The true minimum value of the score function within the parameter space.
+
+    * `noise_model` [NoiseModel]:
+        Noise model to apply to the score.
+
     """
-    def __init__(self, score, space, true_min=None):
+    def __init__(self, score: Callable[..., float], space, true_min=None, noise_model: NoiseModel = ZeroNoise):
```

Comment:
It is not super important, but I would prefer if the default assumption is that noise exists and has to be considered (aka. a ConstantNoise with default values at least). I should hope that our benchmarks use this case more often than the ZeroNoise assumption :)

Reply:
I think it makes sense to remove the default and force users to explicitly decide what they want to do. They have to do so for the space-parameter anyway.